python/MaterialXTest/genshader.py

```diff
@@ -47,6 +47,7 @@ def test_ShaderInterface(self):
 
         # Create a nodedef taking three color3 and producing another color3
         nodeDef = doc.addNodeDef("ND_foo", "color3", "foo")
+        nodeDef.removeOutput("out");
```

Comment:
addNodeDef(...) by default creates an output called "out" which we don't want in this case, so remove it.

Reply:
Would it be possible to use the output "out" below, instead of creating a new output named "o"?

Reply:
Should be possible. I'll just use fooOutput = nodeDef.getOutput("out") instead.